Formula/gcc@11.rb

```diff
@@ -121,6 +122,23 @@ def install
     # Work around GCC install bug
     # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105664
     rm_rf bin.glob("*-gcc-tmp")
+
+    # Add symlinks for libgcc, libgomp, libquadmath and libstdc++ so that bottles
+    # built in CI can find these libraries when using brewed GCC.
+    if OS.linux?
+      lib.install_symlink lib/"gcc/#{version_suffix}/libgcc_s.so"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libgcc_s.a"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libgcc_s.so.1"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libgomp.so"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libgomp.a"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libgomp.so.1"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libquadmath.so"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libquadmath.a"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libquadmath.so.0"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libstdc++.so"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libstdc++.a"
+      lib.install_symlink lib/"gcc/#{version_suffix}/libstdc++.so.6"
+    end
```

Comment:
Creating these symlinks and making them point to the right place seems to be better handled in `brew` than having to rev bump formulae whenever we need to change the preferred GCC version.

Reply:
But if we must handle this in a formula, you probably need `link_overwrite`.

Reply:
Agreed that this feels like a Homebrew/brew thing.

Reply:
I agree. I don't have time myself to look at it until next week though. Feel free to tackle it if anyone has the inclination.

Reply:
We have previously discussed moving this to `brew` so it makes sense to do this now.  We already have `symlink_ld_so` in `extend/os/linux/install.rb` that is called as part of `perform_preinstall_checks`.  I think we can add `symlink_gcc_libs` to this which will make sure the `gcc` runtime libraries are linked whenever installing a formula.

Reply:
@danielnachun opened PR https://github.com/Homebrew/brew/pull/13770
- https://github.com/Homebrew/brew/pull/13770